proto/minder/v1/minder.proto

```diff
@@ -2312,11 +2313,11 @@ message RoleAssignment {
     // subject is the subject to which the role is assigned.
     string subject = 2;
     // display_name is the display name of the subject.
-    string display_name = 5;
+    string display_name = 5 [(buf.validate.field).string.pattern = "^(?!.*<\\/?[a-zA-Z][^>]*>).*$"];
```

Comment:
Rather than trying to exclude patterns (which [OWASP discourages](https://cheatsheetseries.owasp.org/cheatsheets/Input_Validation_Cheat_Sheet.html#allowlist-vs-denylist), let's define an acceptable characters list.  I'd suggest something like:

`[A-Za-z][- [:word:]\[\]\(\)]*`

(Which is a letter, followed by letters, numbers, space, hyphen, underscore and [] and ().)

Reply:
Oh, that's much better! Thank you 💯